bedrock/settings/__init__.py

```diff
@@ -123,6 +143,49 @@
         FLUENT_REPO_PATH,
     ]
 
+    CANONICAL_LOCALES = {
+        # TODO: check whether redundant when we have real translations flowing in to Bedrock
+        "en-US": "en",
+        "en-GB": "en",
+        "en-CA": "en",
```

Comment:
Does this mean we will have 3 locales translated for English but Pocket only uses 1 of them?

Reply:
I may have interpreted `CANONICAL_LOCALES` wrong from the test I looked at, but here am trying to say that all sub-locaes of English get redirected to 'en' - eg https://getpocket.com/en-US/ and https://getpocket.com/en-GB/ etc are 404s, but https://getpocket.com/en/ is where the English locale is served from for Pocket. Am I abusing the setting here @robhudson ?

Reply:
Also, @robhudson I'd welcome your understanding of locale use in Bedrock because I think we have some impending complexity to unpick. 

getpocket.com/es-la/ exists, and so does getpocket.com/es/. 

However it is sounding like /es/ needs to use `es-ES` content from the translation vendor and /es-la/ needs to use `es` from the vendor. This may require some extra wiring, as it feels like it's flipping over the behaviour we have in bedrock and kind of doing the opposite, if you see what I mean. Would def welcome your view!

Reply:
Nope, you're using this as intended. I was getting confused on whether we were actually creating translations for the "key" side of these. Thanks for the clarification.